### ISSUE TITLE
Validate long urls before shortening

### ISSUE THREAD

**Issue by sameshl** [Devsoc-BPGC/Short-Me#18]
If a user tries to shorten a URL to a website which may be along the lines of scam/phishing, etc using ShortMe, Google blocks our website(bp-gc.in) for phishing.

We need to check for any kind such behaviour for any website before we shorten it.

We need to explore any APIs which might add these flagging feature for us.

**Comment by Aviral09** [comment]
https://developers.google.com/safe-browsing/v4 
This article documents some Safe browsing APIs offered by google. Our purpose can be solved by Lookup API.

**Comment by sameshl** [comment]
> https://developers.google.com/safe-browsing/v4
> This article documents some Safe browsing APIs offered by google. Our purpose can be solved by Lookup API.

Yes, this looks right. You can start with this issue if you like

**Comment by Aviral09** [comment]
To work on this further I would need api key generated through the official google developer account for this project.

**Comment by sameshl** [comment]
Hmm, okay.

Is it possible for you to test locally with your own google account and write the relevant code? 

In the end, we could just change the api key.

I don't see any hard dependency on the linked google developer account for this project for this to work locally for you.

**Comment by Aviral09** [comment]
Yeah that can be done. I'll try it out.
